src/osuTK/Input/Key.cs

```diff
@@ -373,6 +364,23 @@ public enum Key : int
         BackSlash,
         /// <summary>The secondary backslash key.</summary>
         NonUSBackSlash,
+
+        // Multimedia keys
+        /// <summary> The mute media key. </summary>
+        Mute,
```

Comment:
let's add them *after* mouse buttons already defined. can be done by changing this line alone.

```suggestion
        Mute = 256,
```